core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceRepository.scala

```diff
@@ -36,12 +58,48 @@ class WorkspaceRepository(dataSource: SlickDataSource) {
       access.workspaceQuery.updateStateWithErrorMessage(workspaceId, state, message)
     }
 
-  def deleteWorkspaceRecord(workspace: Workspace): Future[Boolean] =
+  def deleteWorkspace(workspace: Workspace): Future[Boolean] =
     dataSource.inTransaction { access =>
```

Comment:
You could just all the other signature of `deleteWorkspace` here.

Reply:
I don't understand.

Reply:
You can call the method defined immediately below at line 66, getting the WorkspaceName from the workspace.